Userland/Utilities/test.cpp

```diff
@@ -390,7 +390,7 @@ static OwnPtr<Condition> parse_simple_expression(char* argv[])
         case 'N':
         case 'O':
         case 's':
-            fatal_error("Unsupported operator \033[1m%s", argv[optind]);
+            fatal_error("Unsupported operator \033[1m%s", argv[optind - 1]);
```

Comment:
what? This doesn't seem right, `value` above is also initialized with `argv[optind]`.